menas/src/main/scala/za/co/absa/enceladus/menas/repositories/OozieRepository.scala

```diff
@@ -381,10 +359,8 @@ class OozieRepository @Autowired() (oozieClientRes: Either[OozieConfigurationExc
    */
   private def getCoordinatorFromTemplate(ds: Dataset, wfPath: String): Array[Byte] = {
     val schedule = ds.schedule.get
-    val runtimeParams = schedule.runtimeParams
```

Comment:
I would like to ask particularly @lokm01 if `runtimeParams` and `timezoneOffset ` (couple of lines bellow) not being used is not some omission and it's OK to remove